Libraries/LibWeb/HTML/TraversableNavigable.cpp

```diff
@@ -25,29 +26,35 @@ namespace Web::HTML {
 
 GC_DEFINE_ALLOCATOR(TraversableNavigable);
 
-TraversableNavigable::TraversableNavigable(GC::Ref<Page> page)
-    : Navigable(page)
-    , m_session_history_traversal_queue(vm().heap().allocate<SessionHistoryTraversalQueue>())
+static RefPtr<Gfx::SkiaBackendContext> g_cached_skia_backend_context;
```

Comment:
hmm, have you figured the reason why we allocate >1 traversable on Tweakers? I bet it's SVG images. probably alternative fix here is to check whether we are creating traversable for SVG image and skip context allocation if so. though having a global context cache is probably fine for now.

Reply:
Yeah...
```
> document.querySelectorAll('iframe').length
< 5
> document.querySelectorAll('svg').length
< 141
```

Reply:
> skip context allocation

There's not many situations where we wouldn't already have a context when trying to display an SVG image though?